tutorial/getting_started/install_dependencies.md

```diff
@@ -98,46 +94,28 @@ as well and you can skip to the next section.
 2. We use an environment to encapsulate the Python tools used for this workshop.
    This ensures that the requirements for this workshop do not interfere with
    your other Python projects. To create the environment (named
-   `napari-tutorial`) and install Python 3.9 in it, enter the following command:
+   `image-analysis-23`) and install Python 3.10 in it, enter the following command:
 
     ```bash
-    conda create -n napari-tutorial python=3.9
+    conda env create -f environment.yml
     ```
 
 3. Once the environment setup has finished, activate the environment:
 
     ```bash
-    conda activate napari-tutorial
+    conda activate image-analysis-23
     ```
 
     If you successfully activated the environment, you should now see
-   `(napari-tutorial)` to the left of your command prompt.
-
-4. Install the workshop dependencies with the commands below.
-
-    **If you're on an M1 Mac**:
-
-    ```bash
-    conda install -c conda-forge notebook napari
-    python -m pip install cookiecutter magicgui
-    python -m pip install stardist-napari
-    ```
-
-    Other systems: 
-
-    ```bash
-    conda install -c conda-forge notebook
-    python -m pip install cookiecutter magicgui "napari[all]"
-    python -m pip install stardist-napari
-    ```
+   `(image-analysis-23)` to the left of your command prompt.
 
-5. If you are on a Mac, please install this one additional dependency.
+4. If you are on a Mac, please install this one additional dependency.
```

Comment:
I don't think this is relevant anymore. (Certainly I tried the environment and everything was working fine.)

Reply:
Okay, removed in in https://github.com/scipy-2023-image-analysis/tutorial/pull/5/commits/1811c3f1d16eea72a69ad891873414e37e93642b. @kne42, I understand correctly that your issue in https://github.com/scipy-2023-image-analysis/tutorial/pull/5#discussion_r1248883819 is unrelated to installing `python.app` from conda-forge?